spec/cc/analyzer/issue_spec.rb

```diff
@@ -30,7 +17,7 @@ module CC::Analyzer
       it "adds a fingerprint when it is missing" do
         issue = Issue.new("", output.to_json)
 
-        expect(issue.fingerprint).to eq "433fae1189b03bcd9153dc8dce209fa5"
```

Comment:
These fingerprints were all based on reading the source of `lib/cc/analyzer/config.rb`, which I've deleted, so they changed here because I moved the relevant source to a new fixture file.

I am so mad that anybody ever thought it was a good idea to write tests that relied on an unrelated source file not changing.

Reply:
ohh :(